x/uibc/gmp/README.md

```diff
@@ -0,0 +1,165 @@
+# GMP Handler
+
+Cross chain transfers using Axelar General Message Passing (GMP)
+More info about cosmos gmp find [here](https://docs.axelar.dev/dev/cosmos-gmp)
+
+## Example transfer from fantom evm to umee using Axelar GMP
+
+```bash
+# clone the repo
+$ git clone https://github.com/axelarnetwork/evm-cosmos-gmp-sample.git
+$ cd evm-cosmos-gmp-sample/native-integration/multi-send/solidity
+## Build the contracts on this repo
+$ npm run build
```

Comment:
```suggestion
```bash
# clone the repo
git clone https://github.com/axelarnetwork/evm-cosmos-gmp-sample.git
cd evm-cosmos-gmp-sample/native-integration/multi-send/solidity

## Build the contracts on this repo
npm run build
```